cns/NetworkContainerContract.go

```diff
@@ -66,6 +68,7 @@ const (
 	AzureFirstParty = "AzureFirstParty"
 	KubernetesCRD   = "KubernetesCRD"
 	// TODO: Add OrchastratorType as CRD: https://msazure.visualstudio.com/One/_workitems/edit/7711872
+	KubernetesNodeSubnet = "KubernetesNodeSubnet"
```

Comment:
why do we need this. why can't we reuse `KubernetesCRD`

Reply:
may be we can keep this and remove nodesubnet constant..

Reply:
I had not used CRD because at the end of the day there is no CRD being used here. We can use this string instead of NodeSubnet, but they represent two different things - one is NC type, the other is Orchestrator type. It might be confusing to use the same string for both.

Reply:
Discussed with @tamilmani1989, removing newly introduced constants.